test/specexamples.h

```diff
@@ -641,8 +641,8 @@ const char *ex7_17 =
     "{\n"
     "unquoted : \"separate\",\n"
     "http://foo.com,\n"
-    "omitted value:,\n"
-    ": omitted key,\n"
+    "omitted value: ,\n"
```

Comment:
If you have a closer look at the spec example 7.17, you'll notice that it contains
```
omitted value:°,
```
where the `°` stands for the empty node, so it should not be replaced with a space.

Reply:
ha-ha, yes. And if one is not careless, it's easy to confuse them, just like me.